app/assets/javascripts/client/common_list/EntryCtrl.js

```diff
@@ -0,0 +1,116 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient')
+      .controller('EntryCtrl', EntryCtrl);
```

Comment:
Made a quick look for `EntryCtrl` in this PR but couldn't really tell what it does, the name is not very specific to an implementation so I can assume it's an `abstraction`, however, it does has members like `consensusReportLink` and `assessmentLink` that seem to be very specific to a *concretion* of assessments. maybe this is breaking the DIP principle (?)